### DISCUSSION TITLE
How to whitelist IP address in Rate Limiting?

### DISCUSSION THREAD

**Discussion by HugoSart** [Kong/kong#7590] [Help] [answered]
I can't find anywhere in the documentation if it's possible to whitelist a ipaddress from the limiting. This is something that would be good to have mainly when running production tests.

**Comment by kikito** [comment, marked as answer]
Hello, apologies for not answering sooner.

The ratelimiting plugin doesn't have whitelist capabilities. There are several ways you can go around this problem.

- Identify your users with an authentication plugin (such as key-auth) and apply ratelimiting only to some of them
- Ratelimit one Service, and have a "ulimited Service" with an extra header, or a different path. Possibly protected it with some authentication plugin.
